src/method/method_scan.cpp

```diff
@@ -64,12 +64,14 @@ namespace cubscan
       if (m_list_id == nullptr)
 	{
 	  m_list_id = list_id;
-	  int arg_count = m_list_id->type_list.type_cnt;
-	  m_arg_vector.resize (arg_count);
-	  m_arg_dom_vector.resize (arg_count);
+	  int arg_count = m_arg_count = m_list_id->type_list.type_cnt;
+
+	  m_arg_vector = (DB_VALUE *) db_private_alloc (thread_p, sizeof (DB_VALUE) * arg_count);
+	  m_arg_dom_vector = (TP_DOMAIN **) db_private_alloc (thread_p, sizeof (TP_DOMAIN *) * arg_count);
```

Comment:
```suggestion
	  if (m_arg_vector)
	    {
	  m_arg_vector = (DB_VALUE *) db_private_alloc (thread_p, sizeof (DB_VALUE) * arg_count);
	    }
	  if (m_arg_dom_vector)
	    {
	  m_arg_dom_vector = (TP_DOMAIN **) db_private_alloc (thread_p, sizeof (TP_DOMAIN *) * arg_count);
	    }
```

위와 같이 조건을 추가하여 재할당 되지 않도록 수정하겠습니다.